NEWS.md

```diff
@@ -1,3 +1,11 @@
+# hint 1.73.0
+
+* Add DataExploration header component
+
+# hint 1.72.1
```

Comment:
Where did this news item come from? This change was actually implemented in 1.70.3: https://github.com/mrc-ide/hint/pull/622/files#diff-51920e95310ebfbc1ae31709f3b95f89afffbf4f1a6e38e8b2b406e2fb6197eaL13